libs/ui-toolkit/src/components/trading-button/trading-button.tsx

```diff
@@ -97,7 +97,7 @@ const Content = ({
     {subLabel && (
       <span
         data-sub-label
-        className="mt-0.5 font-mono text-xs leading-tight"
+        className="mt-0.5 font-mono text-xs leading-tight [word-break:break-word]"
```

Comment:
Pretty sure the default TW class `break-all` should work here

Reply:
i can use `break-all` here because it is a number, but i thought it is safer to use `break-word` inside the trading-button content for words as well